src/Components/EditorTransformModel.cs

```diff
@@ -53,6 +54,11 @@ public class EditorTransformModel
         /// </summary>
         public long ComponentId { get; set; }
 
+        /// <summary>
+        ///  Gets or sets a value indicating the component history.
```

Comment:
containing? this is the actual object it seems.

Reply:
ya that's better